iModelCore/BeSQLite/BeSQLite.cpp

```diff
@@ -5598,6 +5602,7 @@ Utf8CP BeSQLiteLib::GetErrorName(DbResult code) {
 +---------------+---------------+---------------+---------------+---------------+------*/
 Utf8CP BeSQLiteLib::GetErrorString(DbResult rc) {
     switch(rc) {
+        case BE_SQLITE_ERROR_DataTransformRequired:         return "failed to import schema as it require data transform";

```

Comment:
failed to import schema: data transformation required

Reply:
```suggestion
        case BE_SQLITE_ERROR_DataTransformRequired:         return "failed to import schema: data transformation required";

```

Reply:
Please approve when you are satisfied.